{"uid":"d6870dbad494f53b","name":"Oops...we found an error preparing this test file: cypress/e2e/features/login.feature","fullName":"Oops...we found an error preparing this test file: cypress/e2e/features/login.feature","time":{"start":1687440636851,"stop":1687440636851,"duration":0},"status":"broken","statusMessage":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\features\\login.feature\n\n","statusTrace":"\n\nError: Build failed with 1 error:\nerror: (3:5): expected: #EOF, #Language, #TagLine, #FeatureLine, #Comment, #Empty, got 'Feature Login page will work depending on the user credentials.'\n    at failureErrorWithLog (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1591:15)\n    at C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1047:28\n    at runOnEndCallbacks (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1463:61)\n    at buildResponseToResult (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1045:7)\n    at C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1157:14\n    at responseCallbacks.<computed> (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:694:9)\n    at handleIncomingPacket (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:749:9)\n    at Socket.readFromStdout (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:670:7)\n    at Socket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Readable.push (node:internal/streams/readable:234:10)\n    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\features\\login.feature\n\n","statusTrace":"\n\nError: Build failed with 1 error:\nerror: (3:5): expected: #EOF, #Language, #TagLine, #FeatureLine, #Comment, #Empty, got 'Feature Login page will work depending on the user credentials.'\n    at failureErrorWithLog (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1591:15)\n    at C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1047:28\n    at runOnEndCallbacks (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1463:61)\n    at buildResponseToResult (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1045:7)\n    at C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:1157:14\n    at responseCallbacks.<computed> (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:694:9)\n    at handleIncomingPacket (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:749:9)\n    at Socket.readFromStdout (C:\\Users\\locar\\OneDrive\\Área de Trabalho\\Auditeste\\Cypress\\ChatGPT-Workshop-Auditeste\\node_modules\\esbuild\\lib\\main.js:670:7)\n    at Socket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Readable.push (node:internal/streams/readable:234:10)\n    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests.","steps":[],"attachments":[{"uid":"a5d5f9f6f06d9491","name":"video recording","source":"a5d5f9f6f06d9491.mp4","type":"video/mp4","size":40028}],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"stepsCount":0},"afterStages":[],"labels":[{"name":"suite","value":"cypress/e2e/features/login.feature"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"d6870dbad494f53b.json","parameterValues":[]}